src/Commands/CreateDraftsMigration.php

```diff
@@ -81,7 +84,10 @@ public function makeClassNameArgument()
         $class_name = join(array_map('ucfirst', explode('_', $class_name)));
         $class_name = "AddNovaDraftsTo{$class_name}";
 
-        if (Schema::hasColumn($this->tableName, 'draft_parent_id')) throw new \Exception("Table '{$this->tableName}' already has drafts");
+        if (Schema::hasColumn($this->tableName, 'draft_parent_id')) {
+            throw new \Exception("Table '{$this->tableName}' already has drafts");
```

Comment:
Nice feature, but i don't think we want to throw an exception here. 

Maybe we should give the user a choice option. For example: `Table users already has drafts, do you wish to continue?`

Reply:
The idea is to remove the exception and put a treatment in order to delete the column and recreate it?

Or the intention is to only inform the user?

Reply:
Only to inform user.

Reply:
```
if ($this->confirm("Table '{$this->tableName}' already has drafts, do you wish to continue?", false)) {
    $this->exit(()
}
```
I had something like this in mind. Inside the if function, instead of the `throw new \Exception`